lib/index.js

```diff
@@ -54,6 +26,7 @@ module.exports = async function () {
   })
 
   const database = new Database({
+    nuxt: this.nuxt,
```

Comment:
Same here, Database should not depend of `nuxt` instance

Reply:
👍